osgp/platform/osgp-core/src/main/resources/db/migration/V20201113200000000__Add_get_outages_to_device_function_mapping.sql

```diff
@@ -0,0 +1,10 @@
+DO
```

Comment:
According to our way of working agreements, ending the version part of Flyway changes with a bunch of zeros should be avoided, to prevent the risk of conflicts.